README.md

```diff
@@ -111,14 +111,18 @@ Sample response:
 {
   "name": "providers/testProvider/vehicles/testVehicle",
   "vehicleState": "ONLINE",
-  "currentTripsIds": ["testTrip"]
+  "currentTripsIds": ["testTrip"],
+  "supportedTripTypes": ["EXCLUSIVE"],
+  "backToBackEnabled": true,
+  "maximumCapacity": 5,
```

Comment:
Nit: Trailing commas are not allowed in JSON.

Reply:
Good point! fixed